src/frontend/src/components/user/getUser.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { UserType } from '../Types';
+
+const baseURL = 'http://127.0.0.1:5000/user';
+
+export default function getInfo(username: string) {
+   const [loading, setLoading] = useState(true);
+   const [error, setError] = useState(false);
+   const [user, setUser] = useState(defaultUser);
+
+   useEffect(() => {
+      //Gets user info on load
+      setLoading(true);
+      setError(false);
+      const CancelToken = axios.CancelToken;
+      const source = CancelToken.source();
+      const timer = setTimeout(() => {
+         axios({
```

Comment:
Do we want a timer here? I would want the page to load immediately

Reply:
Shortened it